packages/react-components/lib/error-overlay.stories.tsx

```diff
@@ -11,13 +11,14 @@ export default {
 
 export const ErrorOverlayPanel: Story = (args) => {
   // override style with userSelect disabled
-  const useStyles = makeStyles(() => ({
+  const useStyles = makeStyles((theme) => ({
     container: {
       display: 'table',
       borderCollapse: 'collapse',
       width: '100%',
       overflowX: 'auto',
       userSelect: 'none',
+      backgroundColor: theme.palette.background.paper,
```

Comment:
We probably shouldn't be specifying this in a story, if this is needed it should be in the component.